stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/call/RtcSession.kt

```diff
@@ -1135,6 +1281,20 @@ public class RtcSession internal constructor(
                         }
 
                         is TrackUnpublishedEvent -> {
+                            // In large calls, TrackUnpublishedEvent may include participant info
+                            // instead of sending separate events
+                            if (event.participant != null) {
+                                logger.i {
+                                    "[TrackUnpublishedEvent] #orphaned-track; Participant info included, " +
+                                        "updating participant for sessionId=${event.sessionId}"
+                                }
+                                val participantState = call.state.getOrCreateParticipant(
```

Comment:
on calling `getOrCreateParticipant` we also end up calling `upsertParticipants` which adds the participant to `internalParticipants` Map and ultimately updates the `_participants` flow - which updatest he public `participants` flow

I think this is not needed as it may un-necessarily send updates to the UI about a participant getting added whereas its just a trackUnpublishedEvent 

We should send an update event only when we have ParticipantAdded/Removed event